cdap-app-templates/cdap-etl/cdap-data-pipeline/src/main/java/co/cask/cdap/datapipeline/LineageOperationsProcessor.java

```diff
@@ -48,14 +52,15 @@
   private final List<String> topologicalOrder;
   private final Dag stageDag;
   // Map of stage to list of operations recorded by that stage
-  private final Map<String, List<Operation>> stageOperations;
+  private final Map<String, List<PipelineOperation>> stageOperations;
   // Map of stage name to another map which contains the output field names to the corresponding origin
   private final Map<String, Map<String, String>> stageOutputsWithOrigins;
   // Set of stages which requires no implicit merge operation for example stages of type join
   private final Set<String> noMergeRequiredStages;
-  private Map<String, co.cask.cdap.api.lineage.field.Operation> processedOperations;
+  private Map<String, Operation> processedOperations;
```

Comment:
Since we change the name of the API, no need to have fully qualified name for the platform api.